Source/Bugsnag.m

```diff
@@ -318,6 +318,11 @@ + (void)setUser:(NSString *_Nullable)userId
     [[self configuration] setUser:userId withName:name andEmail:email];
 }
 
++ (void)removeOnSessionBlock:(BugsnagOnSessionBlock _Nonnull )block
```

Comment:
We need an equivalent `addOnSessionBlock` in Client too.

Reply:
Missed from the underlying task.  Now added.